CONTRIBUTING.md

```diff
@@ -54,9 +54,15 @@ We also follow these project-specific guidelines:
   constructors to all AutoValue classes to prevent classes in other packages
   from extending them.
 
+## Building instrumentation-java
+
+Run `./gradlew clean assemble check` on OS X or Linux, or run
```

Comment:
I just copied the command from .travis.yml.  What is the difference between `build` and `assemble check`?